src/styles/components/search.scss

```diff
@@ -0,0 +1,113 @@
+.search-container {
```

Comment:
It's better not to create deep nested css rules, max 2 levels of deep. Think about refactoring your rules. For example:

Option 1:
.search {
    &-container {}
    &-input {}
    &-input_instruction {}
    &-form {}
    &-form_tip {}
    &-form_top__ {}
}

Option 2:
.search-container {}

.search-input {
     // ...root selector styles
    &_instruction {}
}

.search-form {
   // ...root selector styles
   &_tip {}
   &_tip__any-modificator {}
}

.suggestions-list {
   // ...root selector styles
   &_item {} 
}